openid-4-verifiable-presentations-1_0.md

```diff
@@ -232,9 +232,9 @@ Presentation of Verifiable Credentials using OpenID for Verifiable Presentations
 
 The Authorization Request follows the definition given in [@!RFC6749] taking into account the recommendations given in [@!I-D.ietf-oauth-security-topics].
 
-The Verifier may send an Authorization Request as Request Object by value or by reference as defined in JWT-Secured Authorization Request (JAR) [@RFC9101].
+The Verifier MAY send an Authorization Request as a Request Object either by value or by reference, as defined in the JWT-Secured Authorization Request (JAR) [@RFC9101]. The Request Object MAY contain a subset of parameters, which the Wallet uses to request the creation of a new Request Object from the Verifier. This request is made using an HTTP POST to the Verifier's `request_uri` endpoint. The Wallet MAY support this feature, which involves providing some details about its technical capabilities with the Verifier. This allows the Verifier to generate a Request Object that is specifically designed to match the capabilities of the Wallet making the request. Verifiers that support this feature MUST provide the `request_uri_method`, as defined below, to indicate to the Wallet that they support this feature.
```

Comment:
```suggestion
The Verifier MAY send an Authorization Request as a Request Object either by value or by reference, as defined in the JWT-Secured Authorization Request (JAR) [@RFC9101].

The Wallet may want to provide to the Verifier details about its technical capabilities to
allow the Verifier to generate a request that matches technical capabilities of that Wallet.
To enable this, the first Authorization Request can contain `request_uri_method` parameter
that signals to the Wallet that it can make an HTTP POST to the Verifier's `request_uri`
endpoint with the information about its capabilities. The Wallet MAY continue with JAR
when it receives `request_uri_method` parameter but does not support this new feature.
```
I think it was a following sentence that gave me the impression that request_uri_method cannot be used when request object by reference: `The Request Object MAY contain a subset of parameters, which the Wallet uses to request the creation of a new Request Object from the Verifier.` 
I tried paraphrase/clarify, but I think this needs a separate section tbh.

Reply:
This new wording is much more clear and should be used.